src/core/utils/url.ts

```diff
@@ -33,7 +32,7 @@ export interface URLMeaningfulParts {
   protocol?: string | null;
   slashes?: boolean | null;
   port?: string | null;
-  query: ParsedUrlQuery;
+  query: Record<string, any>;
```

Comment:
`interface ParsedUrlQuery { [key: string]: string | string[]; }`

Is there a reason to use `any` here instead of the previous `string | string[]` ?

Reply:
Agree, `query-string` lib provides `ParsedQuery` type for that